lib/web/servers.go

```diff
@@ -258,10 +258,15 @@ func (h *Handler) getDesktopHandle(w http.ResponseWriter, r *http.Request, p htt
 		return nil, trace.Wrap(err)
 	}
 
+	logins, err := accessChecker.GetAllowedLoginsForResource(windowsDesktops[0])
```

Comment:
Seems we're still calling this on the proxy in the non-unified endpoints, and I think we'll expect the results to still be slightly wrong/the same as what they were right? Are the non-unified endpoints being deprecated or anything?